app/views/planets/show.html.erb

```diff
@@ -6,7 +6,9 @@
        <h5 class="card-text">Price: <%= @planet.price %></h5>
        <h5 class="card-text">Guest number: <%= @planet.number_guests %></h5>
        <%= link_to "Back", planets_path%>
-       <%= link_to "Make a booking", new_planet_booking_path(@planet) %>
+       <% if current_user != @planet.user  %>
```

Comment:
Seems to have a  small problem in indentation 
+ I think a convention is to avoid the "!=" as much as possible for clean code. Maybe we could replace the if with a "unless" condition?